### PR TITLE
ENDOC-72 Add jhipster landing page

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

**Comment by nshaw** [comment]
Completely open to suggestions on better content here. Also visible here right now: https://es-entando.github.io/jhipster.html. 

Side note, I parametrized the special layout with a props option and then toggled the jhipster-specific code with a v-if directive.

**Comment by nshaw** [comment]
@joewhite101 , Bryan Ho provided a couple design options here: https://www.figma.com/proto/tC1kWW05vXXK2LfQYWcHm1/Getting-Started?node-id=797%3A86&viewport=1208%2C-1081%2C0.23255008459091187&scaling=min-zoom. I can shift my initial pass over to match one of those if you have a preference. I think it looks more Entando-y for sure and I'd probably slightly favor the first one.

File: vuepress/docs/.vuepress/components/SpecialLayout.vue [hunk 1]
```diff
@@ -76,7 +76,18 @@
...
     <div class="main-content">
-      <div class="begin-developing layout-container">
+      <div v-if="jhipster" class="layout-container" >
+        <div class="get-started-grid width-container">
+          <div class="get-started-left jhipster">
+            <img src="./assets/logo-jhipster.svg" alt="jhipster"/>
+            JHipster
+          </div>
+          <div class="get-started-right">
+            <p>Greetings, Java Hipster! See the links below to get started with your Entando journey or click <a href="/v6.1/tutorials/backend-developers/generate-microservices-and-micro-frontends.html">here</a> to learn how Entando leverages JHipster to generate micro frontends and microservices.</p>
```
Comment (joewhite101) [inline vuepress/docs/.vuepress/components/SpecialLayout.vue:86]: We'll have to upgrade this to 6.2 at release

Comment (nshaw) [reply vuepress/docs/.vuepress/components/SpecialLayout.vue:86]: Yep, there are several paths like that we'll need to update.

**Pull request merged**